server/config.ts

```diff
@@ -161,6 +161,9 @@ export default {
     mercurySubmit: {
       url: get('MERCURY_SUBMIT_URL', 'http://localhost:3001', requiredInProduction),
     },
+    imsSubmit: {
```

Comment:
Can we change this to manageIntelligence

Reply:
changed to manageIntelligence